one-to-one-sample-app/iOS/SampleApp/MainViewController.m

```diff
@@ -22,7 +22,6 @@ - (void)viewDidLoad {
 - (IBAction)publisherCallButtonPressed:(UIButton *)sender {
     if (self.oneToOneCommunicator.isCallEnabled) {
         [self.mainView callHolderDisconnected];
-        self.oneToOneCommunicator.isCallEnabled = NO;
```

Comment:
Nice, you include this code inside the connect and disconnect methods, really cool